.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "upstream"]
+	path = upstream
+	url = https://github.com/sudo-project/sudo.git
```

Comment:
Direct upstream without mirroring - is this intentional?

Reply:
Yea in the sense that wanted to confirm the submodule approach first before getting a mirror. Could still ask for one.

Reply:
Now having a mirror